docs/usage/self-hosted-configuration.md

```diff
@@ -424,3 +424,13 @@ This is currently applicable to `npm` and `lerna`/`npm` only, and only used in c
 ## username
 
 Mandatory if a GitHub app token is in use using the CLI.
+
+## writeDiscoveredRepos
+
+Optional parameter which allows to write the discovered repositories into a json file instead of renovating them.
```

Comment:
```suggestion
Optional parameter which allows to write the discovered repositories into a JSON file instead of renovating them.
```

Capitalize abbreviation JSON. 😉